automation/pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[build-system]
+requires = [
+    # keep versions in sync with ../pyproject.toml
+    "setuptools==69.0.0",
+    "setuptools_scm==8.0.4",
+    "setuptools_dynamic_dependencies @ git+https://github.com/beeware/setuptools_dynamic_dependencies",
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "x-briefcase-automation"
+description = "A Briefcase plugin for CI automation."
+readme = "README.md"
+license.text = "New BSD"
+classifiers = ["Private :: Do Not Upload"]
```

Comment:
couple notes:

1) Naming the package `x-briefcase-automation` avoids its wheel and sdist from being included in globs for things like `./dist/briefcase-*` and generally helps ensure that you only reference this package's artifacts intentionally and explicitly.

2) Classifier to ensure this package is never accepted by PyPI as an upload.

Reply:
TIL about the Private trove classifier.